banners html (#8949)
braze-inc/braze-docs: _docs/_developer_guide/banner_cards/integration.md

{% tabs %}
{% tab JavaScript %}

> First, you should have a container element for the banner. Ensure you have set its width and height.

Reviewer: ```suggestion
Create a container element for the banner. Be sure to set its width and height.
```

```html
<div id="global-banner-container" style="width: 100%; height: 450px;"></div>
```

> Next, use the [`insertBanner`](https://js.appboycdn.com/web-sdk/latest/doc/modules/braze.html#insertbanner) method to replace the inner HTML of that container element.

Reviewer: ```suggestion
Next, use the [`insertBanner`](https://js.appboycdn.com/web-sdk/latest/doc/modules/braze.html#insertbanner) method to replace the inner HTML of the container element.
```
